### PR TITLE
Not using date objects, but fixed strings with day and month r=@wilsonpage

### PR DESCRIPTION
**Pull request by arcturus**
We were getting a date without time nor tz.

Instead of playing the crazy game of going after the user's tz and rendering in the client. Just got the data coming from sched day and month and that's what we render.

r= @wilsonpage ?

### CODE REVIEW

**Comment by arcturus** [comment]
#Fixes #2

**Comment by wilsonpage** [comment]
@arcturus what format is that date in? Can you give an example string?

**Comment by wilsonpage** [comment]
For the latest design (not yet landed) I wanted '20 May'

**Comment by arcturus** [comment]
Current format we get is '2016-06-14'.

**Comment by arcturus** [comment]
Just changed to '20 May' happy @wilsonpage ?

File: lib/sched.js [hunk 1]
```diff
@@ -104,12 +104,12 @@ Venue.prototype.parseLight = function() {
...
-console.log(string);
-  var days = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
   var months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
 
-  var date = new Date(string);
-  return `${days[date.getDay()]} ${date.getDate()} ${months[date.getMonth()]}`;
+  const components = string.split('-');
+  const month = months[parseInt(components[1]) -1];
+  const day = components[2];
+  return `${month}, ${day}`;
```
Comment (wilsonpage) [inline lib/sched.js]: NIT remove `,`

Comment (wilsonpage) [inline lib/sched.js]: SHould it be

``` js
return `${day} ${month}`;
```

?

Comment (arcturus) [inline lib/sched.js]: Check the latest commit, I did change this already

**Pull request merged**